bench/image-rs/optimized/2s4mh02dvph60euq.ll

```diff
@@ -26352,6 +26356,12 @@ default.unreachable:                              ; preds = %46
   call void @llvm.memcpy.p0.p0.i64(ptr noundef nonnull align 8 dereferenceable(32) %.sroa.311.0..sroa_idx, ptr noundef nonnull align 8 dereferenceable(32) %.sroa.311, i64 32, i1 false)
   br label %65
 
+_ZN5image5color9ColorType15bytes_per_pixel17hcff21ecdd81b7288E.exit.thread: ; preds = %46
+  br label %_ZN5image5color9ColorType13channel_count17h091181b88688e861E.exit
+
+_ZN5image5color9ColorType15bytes_per_pixel17hcff21ecdd81b7288E.exit.thread34: ; preds = %46
+  br label %_ZN5image5color9ColorType13channel_count17h091181b88688e861E.exit
+
```

Comment:
Caused by jump-threading.
Although we have 2 more blocks, we eliminate 2 intermediate phis and redirect more constant phi values into the phi in succ.